src/locales/fr/v2/getting-started.adoc

```diff
@@ -281,15 +282,15 @@ Voici la réponse
 
 . Il manque une parenthèse à la fonction `setTempo()`.
 . Il manque un « e » à la première fonction `fitMedia()`.
-. Il manque une virgule entre le troisième et le quatrième paramètre de la première fonction `fitMedia()`.
+. Il manque une virgule entre le troisième et le quatrième argument de la première fonction `fitMedia()`.
 . Il manque un « M » (majuscule) à la deuxième fonction `fitMedia()`.
-. Dans la seconde fonction `fitMedia()`, l'ordre des paramètres est incorrect : on devrait d'abord retrouver le nom du clip de son, ensuite le numéro de piste.
+. Dans la seconde fonction `fitMedia()`, l'ordre des arguments est incorrect : on devrait d'abord retrouver le nom du clip de son, ensuite le numéro de piste.
```

Comment:
@leaikkache  In the English version, we have standardized on "parameters" instead of arguments here. Does parameters not convey the same meaning in French? I know in English we sometimes use the two interchangeably, which is why we tried to standardize on Parameter for consistency.